pkgs/development/libraries/oracle-instantclient/default.nix

```diff
@@ -22,18 +22,32 @@ let
 
   # determine the version number, there might be different ones per architecture
   version = {
-    x86_64-linux = "19.3.0.0.0";
+    x86_64-linux = "19.16.0.0.0";
+    aarch64-linux = "19.10.0.0.0";
     x86_64-darwin = "19.3.0.0.0";
   }.${stdenv.hostPlatform.system} or throwSystem;
 
+  directory = {
+    x86_64-linux = "1916000";
+    aarch64-linux = "191000";
+    x86_64-darwin = "193000";
+  }.${stdenv.hostPlatform.system} or throwSystem;
+
```

Comment:
```suggestion
```

Reply:
That would be too easy. There's a 0 missing in 191000.

Reply:
ugh.